homeassistant/components/imazu_wall_pad/translations/en.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Please remove the translations folder. All translation strings go into `strings.json`